.github/workflows/minivm-test.yml

```diff
@@ -0,0 +1,32 @@
+name: MiniVM test
+
+on:
+  push:
+    branches: [ "main" ]
+  pull_request:
+    branches: [ "main" ]
+
+
+jobs:
+  build_test:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Repo
+        uses: actions/checkout@v2
+      - name: Install dependencies
+        run: |
+            sudo apt update
+            sudo apt install make gcc qemu wget
+      - name: Set PATH
+        run: echo "export PATH=\$PATH:/usr/local/bin:/opt/clang+llvm-19.1.2-cross-hexagon-unknown-linux-musl/x86_64-linux-gnu/bin" >> ${GITHUB_ENV}
```

Comment:
For some reason this is not effective as of the `make` command below so for now I've fully qualified the path to the compiler.

Reply:
Looks like we should omit the export:

> echo "{environment_variable_name}={value}" >> "$GITHUB_ENV"

https://docs.github.com/en/actions/writing-workflows/choosing-what-your-workflow-does/workflow-commands-for-github-actions#setting-an-environment-variable

Reply:
I thought I'd tried that.  But I'll try again.

Reply:
Yeah - so, perplexingly - making that change causes us to not be able to find `wget` in the `PATH` during the toolchain download step.  

Ah but maybe that just means that now removing `export` made the `PATH` update effective and something about what the `PATH` was updated to is wrong.

Reply:
Oh, interesting. Maybe that `\$PATH` is not expanding as we would expect? Could you echo `\$PATH` and `$PATH`, so that we can see in the log what it is expanding too?

Reply:
Indeed that was it